Signal/src/contact/OWSContactsManager.m

```diff
@@ -360,12 +361,13 @@ + (NSString *)accountLabelForContact:(Contact *)contact recipientId:(NSString *)
     if (phoneNumbersWithTheSameName.count > 1) {
         NSUInteger index =
             [[phoneNumbersWithTheSameName sortedArrayUsingSelector:@selector(compare:)] indexOfObject:recipientId];
+        NSString *indexText = [ViewControllerUtils formatInt:(int)index + 1];
```

Comment:
Fixed a tiny bug here.  If a user had two phone numbers, both "home", they would show up as "home 0" and "home 1".  These indexes should start at 1.